### PR TITLE
🧹 update pubapp

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

The PR cleans up the repo and readme.
It also updates the public app with the new readme and release info.

## Type of change

- [x] documentation update

## How Has This Been Tested?

- [x] New public app passes cwltool validation

**Test Configuration**:
* Environment: cwltool 3.0.20200324120055

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: workflow/kfdrc-somatic-variant-workflow.cwl [hunk 1]
```diff
@@ -39,32 +40,52 @@ doc: |
   There are two WXS only fields `padded_capture_regions` and `unpadded_capture_regions`. If these are not provided in a WXS run,
   the pipeline will fail.
 
-  ### Somatic Variant Calling:
+  ### Standalone Somatic Workflows
```
Comment (migbro) [inline workflow/kfdrc-somatic-variant-workflow.cwl:43]: Can probably remove this from app doc

**Review by migbro — CHANGES_REQUESTED** [review]
The usual app doc vs README, just remove the table from the app doc...unless you want it there to encourage users using the public app, that if they want something less aggressive, to pull from the git repo and use whatever makes sense. That table is really nice, simple, does the job!

File: workflow/kfdrc-somatic-variant-workflow.cwl (hunk 1 above)
Comment (dmiller15) [reply workflow/kfdrc-somatic-variant-workflow.cwl]: Makes sense. Removed.

**Review by migbro — APPROVED** [review]
Good for solving Ops teams issues, file IDs will need updating before app pub

**Comment by dmiller15** [comment]
Yeah if the fix in #65 works I'll close and remake this PR with updated IDs.

**Review by sickler-alex — APPROVED** [review]

**Comment by dmiller15** [comment]
Closing in favor of #66

**Pull request closed without merging**